base/commandLine.cpp

```diff
@@ -68,6 +69,13 @@ static const char HELP_STRING[] =
 	"  -z, --list-games         Display list of supported games and exit\n"
 	"  -t, --list-targets       Display list of configured targets and exit\n"
 	"  --list-saves=TARGET      Display a list of saved games for the game (TARGET) specified\n"
+	"  -a, --add                Add a game from current or specified directory\n"
+	"                           Use --path=PATH before -a, --add to specify a directory.\n"
+	"  --massadd                Add all games from current or specified directory and all sub directories\n"
```

Comment:
> And --add would error out if there are multiple games and no --game option is passed

@tobiatesan I though we agreed during the discussion on IRC today that --add without --game would add all the games found (like the current --massadd)? And that is also what one of your earlier comment above says.